pos_margin/models/pos_order.py

```diff
@@ -0,0 +1,43 @@
+# Copyright (C) 2017 - Today: GRAP (http://www.grap.coop)
+# @author: Sylvain LE GAL (https://twitter.com/legalsylvain)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import api, fields, models
+
+import odoo.addons.decimal_precision as dp
+
+
+class PosOrder(models.Model):
+    _inherit = "pos.order"
+
+    # Columns Section
+    margin = fields.Float(
+        string="Margin",
+        compute="_compute_margin",
+        store=True,
+        digits=dp.get_precision("Product Price"),
```

Comment:
Runbot complaining about:
`2021-07-16 10:09:40,403 160 WARNING openerp_test odoo.addons.base.models.decimal_precision: Deprecated call to decimal_precision.get_precision(<application>), use digits=<application> instead`

Check other lines like this, too

Reply:
Done!